test/End_to_end_test/oidc_v2_test.js

```diff
@@ -144,6 +145,18 @@ var apply_test_parameters = (done) => {
   hybrid_config_with_scope = JSON.parse(JSON.stringify(config_template));
   hybrid_config_with_scope.scope = ['email', 'profile', 'offline_access', 'https://graph.microsoft.com/mail.read'];
 
+  // 6. Hybird flow using client assertion
+  hybrid_config_clientAssertion = JSON.parse(JSON.stringify(hybrid_config));
```

Comment:
hybrid_config is needed since we also have code_config and implicit_config variables, for the three different flows.